governance/SIG Network Charter.md

```diff
@@ -1,97 +1,100 @@
 
 ###Network:
-- Design and maintain client and server protocol communication 
+- Design and maintain client and server protocol communication.
 - Define, publish, and maintain network protocol and data packet design model.
 - Responsible for network presence and state replication.
-- Create and Maintain network simulation tools for testing network condititons.
-
-- Design and maintain multiplayer component architecture supporting client hosted and dedicated server strategies
+- Create and Maintain network simulation tools for testing network conditions.
+- Design and maintain multiplayer component architecture supporting client hosted and dedicated server strategies.
 - Define and maintain specific implementation of multiplayer controller component.
+- Networking specific serialization (as part of the transport layer).
 
 ###Cloud:
-- Design and implement network communication and data exchange for cloud services.
-- Define, standardize, and maintain interface to feature based cloud services.
-- Define and maintain code to enable cloud data services communication
-- Define and implement 3rd party distribution platform services interfaces.
-
-for identity, matchmaking, profiles, persistence.
-
-- Items that are the core responsibilities of SIG
-
-**Cross-cutting Processes**
-- consume and utilize services
-- ensure client and server targets
-- Define communication protocol and stack
-- Define and maintain entity replication network model
-- Responsible for abstraction of console design for dedicated / headless server implementation (core / init)
+- Define, standardize, and maintain interfaces to feature based cloud services.
+- Define and maintain code to enable cloud data services communication.
+
+###Cross-cutting Processes
+- Define client and server build targets for supported platforms, cross-cutting issue with `#sig-core` and `#sig-build`.
+- Define communication protocol and stacks.
+- Define and maintain entity replication network model.
+- Responsible for abstraction of console design for dedicated / headless server implementation (core / init).
 - Define and maintain implementation of communication and data transfer between network based tools.
-
-- Reponsible for design and implementation of controller component until matured and moved into simulation SIG.
-
+- Responsible for design and implementation of controller component until matured and moved into simulation SIG.
 - Publish multiplayer component standard architecture to be shared with SIGs for remote simulation (physics/animation)
-
-- Items that span or require other SIGs or groups and how it relates to this SIG’s responsibilities
+- Define and maintain network layer encryption, as a cross-cutting issue with `#sig-security`.
+- Define and maintain network testing, as a cross-cutting issue with `#sig-testing`.
```

Comment:
Does "network testing" contain testing for the cloud service integration as well?

Reply:
In as much as integrations that owned or related to things owned by the SIG.

Reply:
Added quick note.